pkg/hubble/observer/local_observer_test.go

```diff
@@ -101,23 +103,36 @@ func TestLocalObserverServer_GetFlows(t *testing.T) {
 	go s.Start()
 
 	m := s.GetEventsChannel()
+	input := make([]*observerpb.Flow, numFlows)
+
 	for i := 0; i < numFlows; i++ {
 		tn := monitor.TraceNotifyV0{Type: byte(monitorAPI.MessageTypeTrace)}
 		data := testutils.MustCreateL3L4Payload(tn)
-		m <- &observerTypes.MonitorEvent{
+		event := &observerTypes.MonitorEvent{
 			Timestamp: time.Unix(int64(i), 0),
 			NodeName:  fmt.Sprintf("node #%03d", i),
 			Payload: &observerTypes.PerfEvent{
 				Data: data,
 				CPU:  0,
 			},
 		}
+		m <- event
+		ev, err := pp.Decode(event)
```

Comment:
For what it is worth, you could probably skip this (all flows will look the same) and just check if the timestamps of the `GetFlowResponse` stream are what you expect.